kevinb5617/ResuMake
# Added mockups to docs folder

# Description

Mockups for what we want the website to look like. Outlines both the general ResuMake site that clients would use as well as the personal website that it would generate. These can also be found [here](https://www.figma.com/file/CZNh1TjTK4oLKagSx4gpuP/ResuMake?node-id=0%3A1).

# Related Issue(s) and Pull Request(s)

None

# Solution

I created several mockups [here](https://www.figma.com/file/CZNh1TjTK4oLKagSx4gpuP/ResuMake?node-id=0%3A1) and then downloaded images and added them to the docs folder.

# Steps to Test

1. `git checkout add-mockups-for-pages`
2.  `cd app`
3. `npm install`
4. `npm start`
5. Navigate to 'localhost:3000` in your browser to ensure that nothing broke.
6. Navigate to ResuMake/docs/mockups to view images

# Known Issues

None

# Checklist
